Horse-racing Duals/index.test.js

```diff
@@ -0,0 +1,17 @@
+const index = require('./index');
+const fs = require('fs');
+const simpleCase = fs.readFileSync('./testCaseScenarios/simpleCase.txt', 'utf-8');
+const horseInAnyOrder = fs.readFileSync('./testCaseScenarios/horsesInAnyOrder.txt', 'utf-8');
+const manyHorses = fs.readFileSync('./testCaseScenarios/manyHorses.txt', 'utf-8');
+
+test('Simple case. Smallest difference is 1', () => {
+	expect(index(simpleCase)).toBe(1);
+  });
```

Comment:
Let's read the file in the test case.
```suggestion
test('Simple case. Smallest difference is 1', () => {
    const input = fs.readFileSync('./testCaseScenarios/simpleCase.txt', 'utf-8');
    const output = testSubject(input);
    expect(output).toBe(1);
});
```